### PR TITLE
fix: moving actions that deal with PR preview packages from shared-actions [KHCP-7461]

### PR DESCRIPTION
**Pull request by ValeryG**
Couple of actions used ins both public and private repositories and handling package PR previews. Moved from `Kong/shared-actions`

- also: pr-preview/cleanup added (removes published preview packages) 

Plan to be used in: 
- khcp-ui
- shared-ui-components
- public-ui-components
- konnect-ui-apps
- kongponents
- kong-auth-elements

### CODE REVIEW

File: pr-previews/up-to-date/action.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+name: 'PR Up To Date'
```
Comment (ValeryG) [inline pr-previews/up-to-date/action.yml:1]: this was just moved from shared-actions to here

File: pr-previews/validate/action.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Validate No PR preview references
```
Comment (ValeryG) [inline pr-previews/validate/action.yaml:1]: this was just moved from shared-actions to here

File: pr-previews/cleanup/action.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
+name: Cleanup PR preview packages
```
Comment (adamdehaven) [inline pr-previews/cleanup/action.yml:1]: I don't believe we're using the shared version of this anywhere yet (e.g. some of the component repos)

**Review by adamdehaven — APPROVED** [review]

File: pr-previews/cleanup/action.yml (hunk 3 above)
Comment (ValeryG) [reply pr-previews/cleanup/action.yml:1]: correct - this one is new.

File: pr-previews/README.md
Comment (Drew-Kimberly) [inline pr-previews/README.md:1]: suggestion: might want to consider specifying that these actions are for `PR Package Previews` published from an SDK repo, given the different connotations of the term across Kong

File: pr-previews/cleanup/action.yml [hunk 4]
```diff
@@ -78,5 +78,15 @@ runs:
               echo "Rate limit reached, exiting..."
               exit 1
             fi
+
+            # this is stupid of NPMJS - they do not allow to delete specific version if
```
Comment (zekth) [inline pr-previews/cleanup/action.yml:82]: Please avoid this vocabulary

Comment (ValeryG) [reply pr-previews/cleanup/action.yml]: avoided... :)

File: pr-previews/README.md
Comment (ValeryG) [reply pr-previews/README.md:1]: changed to be more specific

**Review by Drew-Kimberly — APPROVED** [review]

**Comment by ValeryG** [comment]
@Drew-Kimberly , @zekth  please merge this one any time. I have some work waiting for this to be merged and I myself am ... " not authorized to merge this pull request".

**Pull request merged**